controller/TaskQueueWebApi.php

```diff
@@ -24,7 +24,12 @@
 use oat\oatbox\filesystem\FileSystemService;
 use oat\taoTaskQueue\model\TaskLogInterface;
 
-class RestTask extends \tao_actions_CommonModule
+/**
+ * API controller to get task queue data from the WEB.
+ *
+ * @author Gyula Szucs <gyula@taotesting.com>
+ */
+class TaskQueueWebApi extends \tao_actions_CommonModule
```

Comment:
Rename the controller actions can be trick, but the name it's 100% better.
taoNccerDelivery it's using the previous name RestTask for granting rules and url endpoint action.
If we really want to change this and not affect nccer, we can put taoNccerDelivery in manifest that it's requiring taoTaskQueue maximum version 0.5.0 (which it's not ok) OR change also the name of the controller there.

Reply:
good catch, totally forgot. I'll change the name for Nccer.

Reply:
https://github.com/oat-sa/extension-tao-nccer-delivery/pull/372